thumbnail/thumbnail_magick.go

```diff
@@ -209,6 +210,30 @@ func isBMP(bytes []byte) bool {
 	return bytes[0] == 0x42 && bytes[1] == 0x4D
 }
 
+func isHEIC(bytes []byte) bool {
+	// too big ftyp box.
```

Comment:
bytes[0] != 0x52 || bytes[1] != 0x49 || bytes[2] != 0x46 || bytes[2] != 0x46  in WEBP has duplicated `bytes[2] != 0x46 `

Reply:
I created a separate issue.
https://github.com/smartnews/yoya-thumber/issues/25

Reply:
https://github.com/smartnews/yoya-thumber/pull/26 I created the fix for it.

Reply:
thank you so much!